src/components/table/TableCellRenderer.tsx

```diff
@@ -23,25 +23,41 @@ const useStyles = makeStyles((theme: Theme) => ({
       borderBottom: `1px solid ${theme.palette.TwClrBrdrSecondary}`
     }
   },
+  stickyRight: {
+    position: "sticky",
+    right: 0,
+    background: theme.palette.TwClrBg,
+    boxShadow: "-5px 0px 5px grey",
+  },
+  stickyLeft: {
+    position: "sticky",
+    zIndex: 1000,
+    left: 0,
+    background: theme.palette.TwClrBg,
+    boxShadow: "5px 0px 5px grey",
+  }
 }));
 
 export type TableRowType = Record<string, any>;
 
 export default function CellRenderer(props: RendererProps<TableRowType>): JSX.Element {
-  const { column, value, onRowClick, index, className, booleanFalseText, booleanTrueText, editText } = props;
+  const { column, value, onRowClick, index, className, booleanFalseText, booleanTrueText, editText, sticky } = props;
   const id = `row${index}-${column.key}`;
+  const styles = useStyles();
+
+  const classes = className + ' ' + ((sticky === 'left' || sticky === 'both') ? (styles.stickyLeft) : '') + ((sticky === 'right' || sticky === 'both') ? (styles.stickyRight) : '');
```

Comment:
Add a `+ '  '` before the sticky/right check? just in case sticky is 'both', we need a space between styles.stickyLeft and styles.stickyRight.  Or use template string format like in TableHeader.tsx line 58 (which might be cleaner).